stacks-common/src/types/chainstate.rs

```diff
@@ -15,7 +15,7 @@ use crate::consts::{FIRST_BURNCHAIN_CONSENSUS_HASH, FIRST_STACKS_BLOCK_HASH};
 use crate::deps_common::bitcoin::util::hash::Sha256dHash;
 use crate::util::hash::{to_hex, DoubleSha256, Hash160, Sha512Trunc256Sum, HASH160_ENCODED_SIZE};
 use crate::util::secp256k1::{
-    MessageSignature, SchnorrSignature, Secp256k1PrivateKey, Secp256k1PublicKey,
+    MessageSignature, /* SchnorrSignature, */ Secp256k1PrivateKey, Secp256k1PublicKey,
```

Comment:
Why comment this out instead of removing it?  I thought we'd basically agreed to just use the `ThresholdSignature` wrapper around `wsts::common::Signature`?

Reply:
Because I forgot to delete it.  Will remove